### DISCUSSION TITLE
Help on weight for weight

### DISCUSSION THREAD

**Discussion by AzerX2** [codewars/codewars.com#2802] [Q&A] [answered]
Hey,

I don't understand in this exercise why in this test : test.assert_equals(order_weight("2000 10003 1234000 44444444 9999 11 11 22 123"), "11 11 2000 10003 22 123 1234000 44444444 9999")
11 comes before 2000 since it comes after but not 22,
in my code i do from smallest to biggest and when it's equal i do the one that comes last goes first, but it doesn't work for this case
Thank you for your help !

**Comment by EloiseRosen** [comment, marked as answer]
Hi! If you join the Codewars discord and post your question in #help-solve someone can help you there.

**Comment by AzerX2** [comment]
Thanks men
